src/main/java/dev/vality/fraudbusters/mg/connector/converter/FistfulResourceToDomainResourceConverter.java

```diff
@@ -41,6 +39,14 @@ public Resource convert(dev.vality.fistful.base.Resource fistfulResource) {
                         .getDigitalWallet().getPaymentService().getId());
             }
             resource.setDigitalWallet(digitalWallet);
+        } else if (fistfulResource.isSetGeneric()) {
```

Comment:
i think, we should refactor this else-if in future

Reply:
> i think, we should refactor this else-if in future

in future)